src/hooks/api/places.ts

```diff
@@ -102,7 +103,7 @@ const useGetPlacesByCreator = (
       limit: paginationOptions.limit,
       start: paginationOptions.start,
       workflowStatus: 'DRAFT,READY_FOR_VALIDATION,APPROVED,REJECTED',
-      [`sort[${sortOptions.field}]`]: `${sortOptions.order}`,
+      ...createSortingArgument(sortOptions),
```

Comment:
What I meant was, pass down the field + order here and parse it in the hook itself.


```js
sort: {
  field: ...
  order: ...
}
```